lib/datasource/sbt-plugin/index.ts

```diff
@@ -1,12 +1,15 @@
 import { compare } from '../../versioning/maven/compare';
 import { downloadHttpProtocol } from '../maven/util';
 import { parseIndexDir, SBT_PLUGINS_REPO } from './util';
+import { MAVEN_REPO } from '../maven/common';
 import { logger } from '../../logger';
 import { GetReleasesConfig, ReleaseResult } from '../common';
 import { resolvePackageReleases } from '../sbt-package';
 
 export const id = 'sbt-plugin';
 
+export const defaultRegistryUrls = [SBT_PLUGINS_REPO, MAVEN_REPO];
```

Comment:
Is this a change in functionality? Or were we guaranteed that the existing `registryUrls` were always set to `MAVEN_REPO`?

Reply:
Probably, we want to add `MAVEN_REPO` here later for some plugins located in central maven. For now, I've removed it.